integration-tests/util/util.go

```diff
@@ -51,6 +51,11 @@ func WaitForNewPodCreation(clientSet *kubernetes.Clientset, resource interface{}
 				return nil
 			} else if pod.CreationTimestamp.Time.After(startTime) {
 				fmt.Printf("Operator pod %s created after start time but is not in running stage\n", pod.Name)
+				fmt.Println(pod.Status)
```

Comment:
Can these print statements be removed ?

Reply:
Yeah I'll get these debug statements removed